modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/QueriesTest.kt

```diff
@@ -264,31 +266,57 @@ class QueriesTest {
                         .transaction
                         .payload
                         .accountId == ALICE_ACCOUNT_ID
-                }.also {
-                    assert(it)
                 }
+            }.also {
+                assert(it)
             }
     }
 
     @Test
     @WithIroha(AliceHas100XorAndPermissionToBurn::class)
-    @Disabled("Temporarily")
     fun `find permission tokens by account id`(): Unit = runBlocking {
         QueryBuilder.findPermissionTokensByAccountId(ALICE_ACCOUNT_ID)
             .account(ALICE_ACCOUNT_ID)
             .buildSigned(ALICE_KEYPAIR)
             .let { query ->
                 client.sendQuery(query)
-            }.also { tokens ->
+            }.let { tokens ->
                 tokens.any {
                     it.params[ASSET_DEFINITION_PARAM_NAME]
                         ?.cast<Value.Id>()
                         ?.idBox
                         ?.cast<IdBox.AssetDefinitionId>()
                         ?.definitionId == DEFAULT_ASSET_DEFINITION_ID
-                }.also {
-                    assert(it)
                 }
+            }.also {
+                assert(it)
             }
     }
+
+    @Test
+    @WithIroha
+    @Disabled
+    fun `find transaction by hash`(): Unit = runBlocking {
+        val versionedTransaction = TransactionBuilder()
+            .account(ALICE_ACCOUNT_ID)
+            .registerAsset(DefinitionId(randomAlphabetic(10), DEFAULT_DOMAIN_NAME), AssetValueType.Quantity())
+            .buildSigned(ALICE_KEYPAIR)
+        val hash = versionedTransaction.hash()
+
+        client.sendTransaction {
+            versionedTransaction
```

Comment:
method `sendTransaction` returns `CompetableFuture` with hash of transaction. You can initialize transaction inside `sendTransaction {...}` and get hash as a result